deploy/crds/apps.openshift.io_servicebindingrequests_crd.yaml

```diff
@@ -39,6 +39,30 @@ spec:
               description: ApplicationSelector is used to identify the application
                 connecting to the backing service operator.
               properties:
+                bindingPath:
+                  description: BindingPath refers to the path in the application workload's
+                    schema where the binding workload would be referenced.
+                  properties:
+                    customSecretPath:
+                      description: CustomSecret defines the path to a string field
+                        where the secret needs to be assigned.
+                      type: string
+                    podSpecPath:
+                      description: PodSpecPath overrides the default podSpec path
+                      properties:
+                        containers:
+                          description: 'Containers defines the path to the corev1.Containers
+                            reference Example: "spec.template.spec.containers"'
+                          type: string
+                        volumes:
+                          description: 'Volumes defines the path to the corev1.Volumes
+                            reference Example: "spec.template.spec.volumes"'
+                          type: string
+                      required:
+                      - containers
+                      - volumes
```

Comment:
Why volumes is required here?

Reply:
This has been changed to not required. Also now volume has been removed from this PR.